test/distributed/test_c10d_nccl.py

```diff
@@ -253,6 +253,14 @@ def test_init_wo_backend_str(self):
         x = torch.empty(1, device=self.device)
         c10d.all_reduce(x)
 
+    @requires_nccl()
+    @skip_if_lt_x_gpu(1)
+    def test_scalable_init(self):
+        os.environ["TORCH_NCCL_SCALABLE_INIT_RANKS_PER_ROOT"] = "1"
```

Comment:
Should this be cleaned up after the test in case of interference with other tests?

Reply:
also note that this has been changed in the `.cpp` source but it has not been updated in the test

Reply:
Opps sorry.